cpp/ql/lib/semmle/code/cpp/rangeanalysis/RangeAnalysisUtils.qll

```diff
@@ -355,13 +355,13 @@ private predicate linearAccessImpl(Expr expr, VariableAccess v, float p, float q
  * `cmpWithLinearBound(guard, v, Greater(), true)` and
  * `cmpWithLinearBound(guard, v, Lesser(),  false)` hold.
  * If `guard` is `4 - v > 5` then
- * `cmpWithLinearBound(guard, v, Lesser(),  false)` and
- * `cmpWithLinearBound(guard, v, Greater(), true)` hold.
+ * `cmpWithLinearBound(guard, v, Lesser(),  true)` and
+ * `cmpWithLinearBound(guard, v, Greater(), false)` hold.
  *
```

Comment:
Did you test that this change is correct?

Reply:
Yep

Reply:
Quick-eval'ing the predicate on this snippet:
```cpp
int test(int v) {
  if(4 - v > 5) {
    return 1;
  }
  return 0;
}
```
gives: <img width="977" alt="Screenshot 2024-01-09 at 10 24 42" src="https://github.com/github/codeql/assets/4527323/13ef23d3-0c71-40ff-bb71-5b227bc8c2ff">